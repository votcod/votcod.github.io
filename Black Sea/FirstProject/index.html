<!-- Сообщаем браузеру как стоит обрабатывать эту страницу -->
<!DOCTYPE html>
<!-- Оболочка документа, указываем язык содержимого -->
<html lang="ru">
<!-- Заголовок страницы, контейнер для других важных данных (не отображается) -->

<head>
    <!-- Заголовок страницы в браузере -->
    <title>Black Sea</title>
    <!-- Подключаем CSS -->
    <link rel="stylesheet" href="css/style.css">
    <!-- Кодировка страницы -->
    <meta http-equiv="Content-type" content="text/html;charset=UTF-8" />
    <!-- Адаптив -->
    <meta name="viewport" content="width=device-width, initial-scale=1.0,
maximum-scale=1.0, user-scalable=0">
</head>
<!-- Отображаемое тело страницы -->

<body>
    <!-- Оболочка для демонстрации -->
    <div class="wrapper">
        <!-- Контент -->
        <main class="page">
            <header class="header">
                <div class='container'>
                    <div class="header__body">
                        <a href="#" class="header__logo">Black Sea</a>
                        <div class="header__burger">
                            <span></span>
                        </div>
                        <div class="header__menu">
                            <ul class="header__list">
                                <a href="" class="header__link">
                                    <li>Home</li>
                                </a>
                                <a href="" class="header__link">
                                    <li>About me</li>
                                </a>
                                <a href="" class="header__link">
                                    <li>Portfolio</li>
                                </a>
                                <a href="" class="header__link">
                                    <li>Contact me</li>
                                </a>
                            </ul>
                        </div>
                    </div>
                </div>
            </header>
            <div class="main-screen">
                <div class="main-screen__content">
                    <div class='container'>
                        <div class="main-screen__container">
                            <div class="main-screen__slider">
                                <div class="main-screen__slider-item">
                                    <div class="main-screen__title">We need the tonic of wildness</div>
                                    <a href="#" class="main-screen__btn">Read more</a>
                                </div>
                                <div class="main-screen__slider-item">
                                    <div class="main-screen__title">Chop your own wood and it will warm you twice</div>
                                    <a href="#" class="main-screen__btn">Read more</a>
                                </div>
                                <div class="main-screen__slider-item">
                                    <div class="main-screen__title">Life is a series of choices</div>
                                    <a href="#" class="main-screen__btn">Read more</a>
                                </div>
                                <div class="main-screen__slider-item">
                                    <div class="main-screen__title">Love is a friendship set to music</div>
                                    <a href="#" class="main-screen__btn">Read more</a>
                                </div>
                                <div class="main-screen__slider-item">
                                    <div class="main-screen__title">In the middle of difficulty lies opportunity</div>
                                    <a href="#" class="main-screen__btn">Read more</a>
                                </div>
                            </div>
                        </div>
                    </div>
                </div>
                <div class="main-screen__bg ibg">
                    <img src="img/bg.png" alt="" />
                </div>
            </div>
            <div class="learn">
                <div class='container'>
                    <div class="learn__container">
                        <div class="learn__title">At the same time that we are earnest to explore <br />
                            & learn all things</div>
                        <div class="learn__dsc">Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book.</div>
                        <div class="picture__row">
                            <div class="picture__column"><img src="img/bascet.png" alt="" /></div>
                            <div class="picture__column"><img src="img/world.png" alt="" /></div>
                            <div class="picture__column"><img src="img/bag.png" alt="" /></div>
                        </div>
                    </div>
                </div>
            </div>
            <div class="quote">
                <div class='container'>
                    <div class="quote__container">
                        <div class="quote__row">
                            <div class="quote__column">
                                <img src="img/man.png" alt="" />
                            </div>
                            <div class="quote__column">
                                <div class="quote__name">dhavan</div>
                                <div class="quote__icon">“</div>
                            </div>   
                        </div> 
                        <div class="quote__text">
                            <p>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting</p>
                        </div>                       
                    </div>
                </div>
            </div>
            <div class="map">
                <div class='container'>
                    
             
                <div class="map__form-block">
                    <form action="#" method="post" class="map__form">
                        <div class="map__column">
                         <input class="map__form-item" required autocomplete="off" placeholder="Name" tabindex="1" name="username" type="text" />
                         </div>
                         <div class="map__column">
                         <input class="map__form-item" required autocomplete="off" placeholder="Email adress" tabindex="2" name="usermail" type="mailto" />
                         </div>
                         <div class="map__column">
                         <textarea class="map__form-item" tabindex="3" placeholder="Message" rows="4" name="text"></textarea>
                         </div>
                         <div class="map__column">
                         <button  class="map__form-btn"tabindex="4" type="submit"><img src="img/check_mark.png" alt="" /></button>
                         </div>
                    </form>
                </div>
                <div class="map__bg ibg">
                    <img src="img/map.png" alt="" />
                </div> 
                   </div>
            </div>
            <footer class="footer">
                <div class="footer__text">
                    <p>Copyright © 2012 Css Author.com</p>
                </div>
            </footer>
        </main>
        <!-- Подключаем jQuery -->
        <script src="https://code.jquery.com/jquery-3.4.1.min.js"></script>
        <!-- Подключаем JS слайдера -->
        <script src="js/slick.min.js"></script>
        <!-- Подключаем файл JS скриптов -->
        <script src="js/script.js"></script>
    </div>
</body>


</html>