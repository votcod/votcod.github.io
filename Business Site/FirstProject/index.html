<!-- Сообщаем браузеру как стоит обрабатывать эту страницу -->
<!DOCTYPE html>
<!-- Оболочка документа, указываем язык содержимого -->
<html lang="ru">
	<!-- Заголовок страницы, контейнер для других важных данных (не отображается) -->
	<head>
		<!-- Заголовок страницы в браузере -->
		<link rel="stylesheet" href="css/style.css">
		<title>Бизнес сайт</title>
		<!-- Кодировка страницы -->
		<meta http-equiv="Content-type" content="text/html;charset=UTF-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1.0,
maximum-scale=1.0, user-scalable=0">
	</head>
	<body>
		<div class="wrapper">
			<div class="content">
				<header class="header">
					<div class='container'>
						<div class="header__body">
						<a href="#" class="header__logo"><img src="img/header and footer/Logo__header.png" alt="Логотип" /></a>
						<div class="header__burger">
							<span></span>
						</div>
						<div class="header__menu">
							<ul class="header__list">
									<li><a class="header__link" href="#">Home</a></li>
									<li><a class="header__link" href="#">Service</a></li>
									<li><a class="header__link" href="#">About us</a></li>
									<li><a class="header__link" href="#">Pricing Table</a></li>
									<li><a class="header__link" href="#">How it work</a></li>
									<li><a class="header__link" href="#">Happy Clients</a></li>
									<li><a class="header__link" href="#">Contact Us</a></li>
							   </ul>	
						</div>	
					</div>
					</div>
				</header>
				<div class="boost">
					<div class='container'>
						<div class="boost__title">Boost up your Local business</div>
						<div class="boost__subtitle">Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum typesetting</div>
						<div class="boost__form">
							<form action="#" method="post" enctype="multipart/form-data">
								<div class="boost__name">
									<input placeholder="Full Name" required tabindex="1" name="username" type="text" />
								</div>
								<div class="boost__email">
									<input placeholder="Email Adress" required tabindex="2" name="useremail" type="text" />
								</div>
								<div class="boost__phone">
									<input tabindex="3" placeholder="Phone Number" required name="userphone" type="tel" />
								</div>
								<div class="boost__btn-block">
									<button class="boost__btn">Request a quote</button>
								</div>
							</form>	
						</div>
						<div class="boost__image">
							<img src="img/header and footer/Laptop.png" alt="" />
						</div>
						 	
					</div>
				</div>
				<div class="services">
					<div class='container'>
						<div class="services__title">Our Services</div>
						<div class="services__subtitle">Lorem Ipsum is simply dummy Business industry.</div>
						<div class="services__points">
							<div class="services__column">
								<div class="services__item">
									<div class="services__image"><img src="img/services/time__services.png" alt="" /></div>
									<div class="services__body">
										<div class="services__body_title">Font-Awesome Icons</div>
										<div class="services__body_text"><p>Lorem Ipsum is simply dummy text of the printing and typesetting ind
                                        ustry. Lorem Ipsum has been the industry's standard dummy text ever 
                                        since the 1500s, when an unknown printer.</p></div>
									</div>	
								</div>
							</div>
							<div class="services__column">
								<div class="services__item">
									<div class="services__image"><img src="img/services/gear__services.png" alt="" /></div>
									<div class="services__body">
										<div class="services__body_title">Fully Responsive Design</div>
										<div class="services__body_text">Many desktop publishing packages and web page editors now use Lorem Ipsum as their default model text, and a search for 'lorem ipsum' will uncover many web sites still in their infancy</div>
									</div>	
								</div>
							</div>
							<div class="services__column">
								<div class="services__item">
									<div class="services__image"><img src="img/services/study__services.png" alt="" /></div>
									<div class="services__body">
										<div class="services__body_title">Working Contact form</div>
										<div class="services__body_text">Many desktop publishing packages and web page editors now use Lorem Ipsum as their default model text, and a search for 'lorem ipsum' will un
                                        cover many web sites still in their infancy</div>
									</div>	
								</div>
							</div>
							<div class="services__column">
								<div class="services__item">
									<div class="services__image"><img src="img/services/rocket__services.png" alt="" /></div>
									<div class="services__body">
										<div class="services__body_title">Retina Ready icons here</div>
										<div class="services__body_text">There are many variations of passages of Lorem Ipsum available, but the majority have suffered alteration in some form, by injected humour
                                        or randomised words which don't look even slightly believable.
                                        </div>
									</div>	
								</div>
							</div>
							<div class="services__column">
								<div class="services__item">
									<div class="services__image"><img src="img/services/friendly__services.png" alt="" /></div>
									<div class="services__body">
										<div class="services__body_title">Friendly Code</div>
										<div class="services__body_text">Finibus Bonorum et Malorum" by Cicero are also reproduced in their 
                                        exact original form, accompanied by English versions from the 1914 translation by H. Rackham</div>
									</div>	
								</div>
							</div>
							<div class="services__column">
								<div class="services__item">
									<div class="services__image"><img src="img/services/dollar__services.png" alt="" /></div>
									<div class="services__body">
										<div class="services__body_title">Flat trendy landing page Design</div>
										<div class="services__body_text">Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since 
                                        the 1500s, when an unknown printer.</div>
									</div>	
								</div>
							</div>
						</div>
					</div>
				</div>
				<div class="about">
					<div class="about-flex">
						<div class="about-flex__image"><!-- <img src="img/clients/picture__about.png" alt="" /> --></div>
						<div class="about-flex__content">
							<div class="about-flex__body">
							<div class="about-flex__title">About us</div>
							<div class="about-flex__text"><p>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the ore recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum</p></div>
							<div class="about-flex__btn-block"><a href="" class="about-flex__btn">Read more</a></div>
							
						</div>
						</div>
					</div>
				</div>
				<div class="price">
					<div class='container'>
						<div class="price__title">Choos your price</div>
						<div class="price__subtitle">Lorem Ipsum is simply dummy text of the printing typesetting</div>
					
					<div class="price-flex">
						<div class="price-flex__column">
							<div class="price-flex__item">
								<div class="price-flex__lable">Basic plan</div>
								<div class="price-flex__value"><span><sup>$</sup></span>22</div>
								<div class="price-flex__value_title">per month</div>
								<div class="price-flex__performance">
									<ul>
										<li>1 Account</li>
										<li>8 hour support</li>
										<li>Custom Cloud Services</li>
									</ul>
								</div>
								<div class="price-flex__btn-block">
									<a href="#" class="price-flex__btn">Get Started</a></div>
								</div>
							</div>
							<div class="price-flex__column price-flex__column_optimal">
							<div class="price-flex__item price-flex__item_optimal">
								<div class="price-flex__lable price-flex__lable_optimal">Premium plan</div>
								<div class="price-flex__value price-flex__value_optimal"><span><sup>$</sup></span>42</div>
								<div class="price-flex__value_title price-flex__value_optimal-title">per month</div>
								<div class=" price-flex__performance price-flex__performance_optimal">
									<ul>
										<li>2 Account</li>
										<li>100MB Storage</li>
										<li>Custom Cloud Services</li>
									</ul>
								</div>
								<div class="price-flex__btn-block price-flex__btn-block_optimal">
									<a href="#" class=" price-flex__btn price-flex__btn_optimal">Get Started</a></div>
								</div>
							</div>
							<div class="price-flex__column">
							<div class="price-flex__item">
								<div class="price-flex__lable">Gold plan</div>
								<div class="price-flex__value"><span><sup>$</sup></span>62</div>
								<div class="price-flex__value_title">per month</div>
								<div class="price-flex__performance">
									<ul>
										<li>8 Account</li>
										<li>24 hour support</li>
										<li>Custom Cloud Services</li>
									</ul>
								</div>
								<div class="price-flex__btn-block">
									<a href="#" class="price-flex__btn">Get Started</a></div>
								</div>
							</div>
						</div>
					  </div>
					</div>
				</div>
				<div class="how">
					<div class='container'>
						<div class="how__title">How it works</div>
						<div class="how__subtitle">Lorem Ipsum is simply dummy Business industry</div>
						<div class="how-flex">
							<div class="how-flex__column">
								<div class="how-flex__item">
									<div class="how-flex__image"><img src="img/how/headphones__how.png" alt="" /></div>
									<div class="how-flex__title">Step 1</div>
									<div class="how-flex__text"><p>Lorem Ipsum is simply dummy text of the printing and typesetting ind
                                    ustry. Lorem Ipsum has been</p></div>
								</div>
							</div>
							<div class="how-flex__column">
								<div class="how-flex__item">
									<div class="how-flex__image"><img src="img/how/lock__how.png" alt="" /></div>
									<div class="how-flex__title">Step 2</div>
									<div class="how-flex__text"><p>Lorem Ipsum is simply dummy text of the printing and typesetting ind
                                    ustry. Lorem Ipsum has been</p></div>
								</div>
							</div>
							<div class="how-flex__column">
								<div class="how-flex__item">
									<div class="how-flex__image"><img src="img/how/cloud__how.png" alt="" /></div>
									<div class="how-flex__title">Step 3</div>
									<div class="how-flex__text"><p>Lorem Ipsum is simply dummy text of the printing and typesetting ind
                                    ustry. Lorem Ipsum has been</p></div>
								</div>
							</div>
							<div class="how-flex__column">
								<div class="how-flex__item">
									<div class="how-flex__image"><img src="img/how/like__how.png" alt="" /></div>
									<div class="how-flex__title">Step 4</div>
									<div class="how-flex__text"><p>Lorem Ipsum is simply dummy text of the printing and typesetting ind
                                    ustry. Lorem Ipsum has been</p></div>
								</div>
							</div>							
						</div>
						<div class="how__btn-block">
							<a href="#" class="how__btn">Get start Now</a>
						</div>	
					</div>
				</div>
				<div class="clients">
					<div class='container'>
						<div class="clients__title">Happy Clients</div>
						<div class="clients__subtitle">We are explain who is using our business solutions</div>
						<div class="clients-flex">
							<div class="clients-flex__column">
								<div class="clients-flex__item">
									<div class="clients-flex__image"><img src="img/clients/people1__clients.png" alt="" /></div>
									<div class="clients-flex__body">
										<div class="clients-flex__text"><p><span>"</span>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make.<span>"</span></p></div>
										<div class="clients-flex__name"><span>Darwin Michle, </span>  Project manager</div>
									</div>
								</div>
							</div>
							<div class="clients-flex__column">
								<div class="clients-flex__item">
									<div class="clients-flex__image"><img src="img/clients/people2__clients.png" alt="" /></div>
									<div class="clients-flex__body">
										<div class="clients-flex__text"><p><span>"</span>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrystandard dummy text ever since the 1500s, wen an unknown printer took a galley of type and scrambled to make a type specimen book.<span>"</span></p></div>
										<div class="clients-flex__name"><span>Madam Elisabath, </span>  Creative Director</div>
									</div>
								</div>
							</div>
							<div class="clients-flex__column">
								<div class="clients-flex__item">
									<div class="clients-flex__image"><img src="img/clients/people3__clients.png" alt="" /></div>
									<div class="clients-flex__body">
										<div class="clients-flex__text"><p><span>"</span>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make.<span>"</span></p></div>
										<div class="clients-flex__name"><span>Clips arter, </span>  Lipsum directer</div>
									</div>
								</div>
							</div>
							<div class="clients-flex__column">
								<div class="clients-flex__item">
									<div class="clients-flex__image"><img src="img/clients/people4__clients.png" alt="" /></div>
									<div class="clients-flex__body">
										<div class="clients-flex__text"><p><span>"</span>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrystandard dummy text ever since the 1500s, wen an unknown printer took a galley of type and scrambled to make a type specimen book.<span>"</span></p></div>
										<div class="clients-flex__name"><span>zam cristafr, </span>  manager</div>
									</div>
								</div>
							</div>
						</div>
					</div>
				</div>
				<div class="letter">
					<div class='container'>
						<div class="letter__title">newsletter</div>
						<div class="letter__subtitle">Lorem Ipsum is simply dummy Business industry</div>
						<div class="letter__form">
							<form action="#" method="post" enctype="multipart/form-data">
								<div class="letter__wrapper1">
								<div class="letter__name">
									<input placeholder="Name" required tabindex="1" name="username" type="text" />
								</div>
								</div>
								<div class="letter__wrapper2">
								<div class="letter__email">
									<input placeholder="Email Adress" required tabindex="2" name="useremail" type="text" />
								</div>
								</div>
								<div class="letter__wrapper3">
								<div class="letter__btn-block">
									<button class="letter__btn">Subscribe now</button>
								</div>
								</div>
							</form>
						</div>
						<div class="letter__contacts-block">
							<div class="letter__contacts">
								<div class="letter__contacts_title">Address</div>
								<div class="letter__list">
									<ul>
										<li>Business street</li>
										<li>Second north section</li>
										<li>Bavani nagar. 88769</li>
										<li>Call : +987-9976-999</li>
										<li><span>info@businessface.com</span></li>
									</ul>
								</div>
							</div>
							<div class="letter__logo"><img src="img/letter/Logo.png" alt="" /></div>
						</div>
						<div class="letter__menu">
							<ul>
								<li><a href="#">Home</a></li>
								<li><a href="#">Service</a></li>
								<li><a href="#">About us</a></li>
								<li><a href="#">Pricing Table</a></li>
								<li><a href="#">How it work</a></li>
								<li><a href="#">Happy Clients</a></li>
								<li><a href="#">Contact Us</a></li>
							</ul>
						</div>
						<div class="letter__phone-block"><div class="letter__phone"> +987 9976 999</div></div>
						<div class="letter__social-icons-block"><div class="letter__social-icons">
							<ul>
								<li><a href="#"><img src="img/letter/twitter-icon__letters.png" alt="" /></a></li>
								<li><a href="#"><img src="img/letter/facebook-icon__letter.png" alt="" /></a></li>
								<li><a href="#"><img src="img/letter/google-icon__letter.png" alt="" /></a></li>
								<li><a href="#"><img src="img/letter/linkedin-icon__letter.png" alt="" /></a></li>
							</ul>
						</div>
						</div>	
					</div>
				</div>
			</div>
			<footer class="footer">
				<div class='container'>
					<div class="footer__text">
						<p>All contents © copyright 2014 Business Theme. All rights reserved Designed by : akhilwebfolio</p>
					</div>	
				</div>
			</footer>

			<script src="https://code.jquery.com/jquery-3.4.1.min.js"></script>
			<script src="js/script.js"></script>
		</div>
	</body>
	</html>