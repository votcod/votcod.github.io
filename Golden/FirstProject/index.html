<!-- Сообщаем браузеру как стоит обрабатывать эту страницу -->
<!DOCTYPE html>
<!-- Оболочка документа, указываем язык содержимого -->
<html lang="ru">
<!-- Заголовок страницы, контейнер для других важных данных (не отображается) -->

<head>
    <!-- Заголовок страницы в браузере -->
    <title>Название страницы</title>
    <!-- Подключаем CSS -->
    <link rel="stylesheet" href="css/style.css">
    <!-- Кодировка страницы -->
    <meta http-equiv="Content-type" content="text/html;charset=UTF-8" />
    <!-- Адаптив -->
    <meta name="viewport" content="width=device-width, initial-scale=1.0,
maximum-scale=1.0, user-scalable=0">
</head>
<!-- Отображаемое тело страницы -->

<body>
    <!-- Оболочка для демонстрации -->
    <div class="wrapper">
        <!-- Контент -->
        <main class="page">
            <header class="header">
                <div class='container'>
                    <div class="header__body">
                        <a href="#" class="header__logo"><img src="img/logo.png.png" alt="" /></a>
                        <div class="header__burger">
                            <span></span>
                        </div>
                        <div class="header__menu">
                            <ul class="header__list">
                                <li><a class="header__link" href="#">Home</a></li>
                                <li><a class="header__link" href="#">Services</a></li>
                                <li><a class="header__link" href="#">Portfolio</a></li>
                                <li><a class="header__link" href="#">About</a></li>
                                <li><a class="header__link" href="#">Contact</a></li>
                            </ul>
                        </div>
                    </div>
                </div>
            </header>
            <div class="main-screen">
                <div class="main-screen__content">
                    <div class='container'>
                        <div class="main-screen__title">Welcome To Our Studio!</div>
                        <div class="main-screen__big-title">It’s nice to Meet you</div>
                        <a href="#" class="main-screen__button">Tell Me More</a>
                    </div>
                </div>
                <div class="main-screen__bg ibg">
                    <img src="img/first-bg.png" alt="" />
                </div>
            </div>
            <div class="content">
                <div class="services">
                    <div class='container'>
                        <div class="services__title title">Services</div>
                        <div class="services__subtitle subtitle">Proin iaculis purus consequat sem cure. </div>
                        <div class="services__row">
                            <div class="services__column">
                                <div class="services__item">
                                    <div class="services__image"><img src="img/Services/icon1.png.png" alt="" /></div>
                                    <div class="services__heading">E-Commerce</div>
                                    <div class="services__text">
                                        <p>Proin iaculis purus consequat sem cure
                                            digni ssim. Donec porttitora entum suscipit
                                            aenean rhoncus posuere odio in tincidunt.</p>
                                    </div>
                                </div>
                            </div>
                            <div class="services__column">
                                <div class="services__item">
                                    <div class="services__image"><img src="img/Services/icon2.png.png" alt="" /></div>
                                    <div class="services__heading">Responsive Web</div>
                                    <div class="services__text">
                                        <p>Proin iaculis purus consequat sem cure
                                            digni ssim. Donec porttitora entum suscipit
                                            aenean rhoncus posuere odio in tincidunt.</p>
                                    </div>
                                </div>
                            </div>
                            <div class="services__column">
                                <div class="services__item">
                                    <div class="services__image"><img src="img/Services/icon3.png.png" alt="" /></div>
                                    <div class="services__heading">Web Security</div>
                                    <div class="services__text">
                                        <p>Proin iaculis purus consequat sem cure
                                            digni ssim. Donec porttitora entum suscipit
                                            aenean rhoncus posuere odio in tincidunt.</p>
                                    </div>
                                </div>
                            </div>
                        </div>
                    </div>
                </div>
                <div class="portfolio">
                    <div class='container'>
                        <div class="portfolio__title title">Our portfolio</div>
                        <div class="portfolio__subtitle subtitle">Proin iaculis purus consequat sem cure.</div>
                        <div class="portfolio__row">
                            <div class="portfolio__column">
                                <img src="img/Portfolio/img1.jpg.png" alt="" />
                            </div>
                            <div class="portfolio__column">
                                <img src="img/Portfolio/img2.jpg.png" alt="" />
                            </div>
                            <div class="portfolio__column">
                                <img src="img/Portfolio/img3.jpg.png" alt="" />
                            </div>
                        </div>
                    </div>
                </div>
                <div class="about">
                    <div class='container'>
                        <div class="about__title title">About Us</div>
                        <div class="about__subtitle subtitle">Proin iaculis purus consequat sem cure.</div>
                        <div class="about__column">
                            <div class="about__row">
                                <div class="about__body">
                                    <div class="about__date">July 2010</div>
                                    <div class="about__heading">Our Humble Beginnings</div>
                                    <div class="about__text">
                                        <p>Proin iaculis purus consequat sem cure
                                            digni ssim. Donec porttitora entum suscipit
                                            aenean rhoncus posuere odio in tincidunt. Proin
                                            iaculis purus consequat sem cure digni
                                            ssim. Donec porttitora entum suscipit.</p>
                                    </div>
                                </div>
                                <div class="about__image"><img src="img/About/about1.jpg.png" alt="" /></div>
                            </div>
                            <div class="about__row about__row_right">
                                <div class="about__image"><img src="img/About/about2.jpg.png" alt="" /></div>
                                <div class="about__body about__body_right">
                                    <div class="about__date">January 2011</div>
                                    <div class="about__heading">Facing Startup Battles</div>
                                    <div class="about__text">
                                        <p>Proin iaculis purus consequat sem cure
                                            digni ssim. Donec porttitora entum suscipit
                                            aenean rhoncus posuere odio in tincidunt. Proin
                                            iaculis purus consequat sem cure digni
                                            ssim. Donec porttitora entum suscipit aenean
                                            rhoncus posuere odio in tincidunt.</p>
                                    </div>
                                </div>
                            </div>
                            <div class="about__row">
                                <div class="about__body">
                                    <div class="about__date">December 2012</div>
                                    <div class="about__heading">Enter The Dark Days</div>
                                    <div class="about__text">
                                        <p>Proin iaculis purus consequat sem cure
                                            digni ssim. Donec porttitora entum suscipit
                                            aenean rhoncus posuere odio in tincidunt. Proin
                                            iaculis purus consequat sem cure digni.</p>
                                    </div>
                                </div>
                                <div class="about__image"><img src="img/About/about3.jpg.png" alt="" /></div>
                            </div>
                            <div class="about__row about__row_right">
                                <div class="about__image"><img src="img/About/about4.jpg.png" alt="" /></div>
                                <div class="about__body about__body_right">
                                    <div class="about__date">february 2014</div>
                                    <div class="about__heading">Our Triumph</div>
                                    <div class="about__text">
                                        <p>Proin iaculis purus consequat sem cure
                                            digni ssim. Donec porttitora entum suscipit
                                            aenean rhoncus posuere odio in tincidunt. Proin
                                            iaculis purus consequat sem cure digni
                                            ssim. Donec porttitora entum suscipit aenean.</p>
                                    </div>
                                </div>
                            </div>
                        </div>
                    </div>
                </div>
            </div>
            <footer class="footer">
                <div class='container'>
                    <div class="footer__text">© Copyright 2014 FreebiesXpress.com</div>
                </div>
            </footer>
        </main>
        <!-- Подключаем jQuery -->
        <script src="https://code.jquery.com/jquery-3.4.1.min.js"></script>
        <!-- Подключаем файл JS скриптов -->
        <script src="js/script.js"></script>
    </div>
</body>

</html>